src/main/java/org/terasology/ShatteredPlanes/Rasterizer/EasterEggRasterizer.java

```diff
@@ -51,15 +51,15 @@ public void initialize() {
     public void generateChunk(CoreChunk chunk, Region chunkRegion) {
         SurfacesFacet surfacesFacet = chunkRegion.getFacet(SurfacesFacet.class);
         EasterEggFacet eggFacet = chunkRegion.getFacet(EasterEggFacet.class);
-        for (Vector3i position : chunkRegion.getRegion().expand(-eggRadius - 1)) {
+        for (Vector3ic position : chunkRegion.getRegion().expand(new Vector3i( -eggRadius - 1))) {
```

Comment:
This will modify the `chunkRegion` in-place - I'm not sure whether that is desired here 🤔 Previously, `expand` created a new region, e.g.:
```suggestion
        for (Vector3ic position : new BlockRegion(chunkRegion.getRegion()).expand(new Vector3i( -eggRadius - 1))) {
```

Reply:
I'm wondering how cases like this we have, i.e., how many unknown bugs we introduce 🙈

Reply:
guess we'll see in the playtest